src/views/siteGridReference.html

```diff
@@ -26,7 +26,7 @@
           <summary><span id="site-grid-reference-summary" class="summary">Help to find the grid reference</span></summary>
           <div class="panel panel-border-narrow">
             <p>
-              Use the free <a id="site-grid-reference-finder-link" href="http://gridreferencefinder.com/" target="_blank" rel="noopener noreferrer">UK Grid Reference Finder (opens new tab)</a>.
+              Use the free <a id="site-grid-reference-finder-link" href="http://gridreferencefinder.com/" target="_blank" rel="noopener noreferrer">UK Grid Reference Finder (opens new window)</a>.
```

Comment:
Martin tells me he made a design mistake here and that (opens new window) should stay as (opens new tab)
(Evidently he says it wasn't really him but instead another boy but he's run off.......)

Reply:
Reverted